### ISSUE TITLE
php artisan vendor:publish command says "Unable to locate publishable resources."

### ISSUE THREAD

**Issue by commadelimited** [aws/aws-sdk-php-laravel#214]
### Describe the bug

I have a Lumen 8.0 app into which I'm trying to install this SDK.

1) I install via composer: `composer require aws/aws-sdk-php-laravel`
2) I add this line (`$app->register(Aws\Laravel\AwsServiceProvider::class);`) to `bootstrap/app.php`
3) I confirm I have the proper environment variables in place
4) I run `php artisan vendor:publish  --provider="Aws\Laravel\AwsServiceProvider"`
5) I get this output: `Unable to locate publishable resources. Publishing complete.`

I tried manually importing that class:
```
use Aws\Laravel\AwsServiceProvider;
$app->register(AwsServiceProvider::class);
// and nothing
```

I tried running `composer dumpautoload` and then the vendor command and nothing.

I've found posts on Stack Overflow which suggest running the `php artisan vendor:publish` alone. That command output does not include anything related to AWS.

I even tried this suggestion found on the Lumen Vendor Publish package:
https://github.com/laravelista/lumen-vendor-publish/issues/16#issuecomment-917424678

Can you offer any insights into why this might be failing? I suspect I could simply copy `vendor/aws/aws-sdk-php-laravel/config/aws_publish.php` into my `config/` directory as `aws.php` but given you have specific instructions for Lumen I thought I'd ask here first.

### Expected Behavior

I expected the file `config/aws.php` to be created as per the instructions in the readme.

### Current Behavior

When I run this command:
```
$ php artisan vendor:publish  --provider="Aws\Laravel\AwsServiceProvider"
```

I get the following output:
```
Unable to locate publishable resources.
Publishing complete.
```

### Reproduction Steps

1) I install via composer: `composer require aws/aws-sdk-php-laravel`
2) I add this line (`$app->register(Aws\Laravel\AwsServiceProvider::class);`) to `bootstrap/app.php`
3) I confirm I have the proper environment variables in place
4) I run `php artisan vendor:publish  --provider="Aws\Laravel\AwsServiceProvider"`
5) I get this output: `Unable to locate publishable resources. Publishing complete.`

### Possible Solution

It appears that the publish command can't find the package, but it's clearly there when viewing the vendor folder.

### SDK version used

3.7

### Environment details (OS name and version, etc.)

Mac OS 10.15.7 (19H1615)

**Comment by yenfryherrerafeliz** [comment]
Hi @commadelimited, sorry for the delay on this. Can you please confirm if you are still having this issue so we can assist you further?

Thanks!

**Comment by Shrikant-28** [comment]
Hi @commadelimited,
I have tried another way.
I have registered "**AwsServiceProvider**" inside config/app.php in '**Autoloaded Service Providers**' section
`Aws\Laravel\AwsServiceProvider::class,`

After registering using `composer dump-autoload command`

If want to check publishable assets use command  `php artisan vendor:publish`

You will see  
Provider: Aws\Laravel\AwsServiceProvider ....................................................................................................... 1
and 
 Tag: aws-config ............................................................................................................................................................ 10

Type the sequence number of Provider or Tag and hit enter.
You will see aws.php file copied inside config folder.
 
![image](https://github.com/aws/aws-sdk-php-laravel/assets/59720723/2f74772a-495d-4ac5-84fa-c00414c41d83)
![image](https://github.com/aws/aws-sdk-php-laravel/assets/59720723/e3975e4b-bb20-4170-943c-6434a6f121c9)
